.github/workflows/lighthouse.yml

```
@@ -0,0 +1,37 @@
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Build application
+        run: npm run build
+
+      - name: Run Lighthouse CI
+        uses: treosh/lighthouse-ci-action@v12
+        with:
+          configPath: './lighthouserc.json'
+          uploadTargets: 'temporary-public-storage'
```

actionlint (1.7.12)

[error] input "uploadTargets" is not defined in action "treosh/lighthouse-ci-action@v12". available inputs are "artifactName", "basicAuthPassword", "basicAuthUsername", "budgetPath", "configPath", "runs", "serverBaseUrl", "serverToken", "temporaryPublicStorage", "uploadArtifacts", "uploadExtraArgs", "urls"

(action)